dist/vue-ics.common.js

```diff
@@ -1,13 +1,13 @@
 /*!
- * vue-ics v0.1.3 
- * (c) 2020 Stanislav Mihaylov <ceo@pepper.llc>
+ * vue-ics v0.1.5 
+ * (c) 2022 Inventive Group
```

Comment:
woah, we probably need to keep Stanislav's name around.

Reply:
this file is auto generated. i changed this in the package.json to update it here. i agree it's probably better to give the original author credit.